packages/pn-commons/src/components/CustomDatePicker.tsx

```diff
@@ -1,12 +1,16 @@
+/* eslint-disable sonarjs/no-identical-functions */
 import { de, enGB, fr, it, sl } from 'date-fns/locale';
```

Comment:
Create two test cases that check the two views

Reply:
done